api/src/sessionPhase1/import/sessionPhase1ImportService.ts

```diff
@@ -105,11 +105,15 @@ const createSession = async (
   foundCohort: CohortDocument,
 ): Promise<SessionCohesionCenterImportReport> => {
   let warning = "";
-  if (sessionCenter.sessionPlaces > sessionCenter.cohesionCenterPlacesTotal) {
-    logger.warn(
-      `Session with wrong place number (${sessionCenter.sessionPlaces} > ${sessionCenter.cohesionCenterPlacesTotal}) center ${foundCenter.matricule} and cohort ${foundCohort.snuId}`,
-    );
-    warning = "session places > center places";
+  if (sessionCenter.sessionPlaces > foundCenter.placesTotal || !foundCenter.placesTotal) {
```

Comment:
```suggestion
  if (sessionCenter.sessionPlaces > foundCenter.placesTotal!) {
```
cf autre com, plus fix type